qiskit/pulse/ir/ir_instructions.py

```diff
@@ -0,0 +1,335 @@
+# This code is part of Qiskit.
```

Comment:
There are already several `Instruction` types in Qiskit, one of them in Pulse. On the one hand it might be more confusing, on the other hand, we already resolved all of the ambiguous referencing issues. @nkanazawa1989 , what do you think?